.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+Our [Code contribution guidline](https://discuss.neos.io/t/code-contribution-guideline/503) might also be helpful to start with.
+
+## Improve
+
+Your are a developer, a design or an UX expert and want to improve the products, what's the next step?
+
+## You are a design or UX expert ?
+
+The Neos project is focused on providing first class UX for our products. Help out with Neos CMS user interaction design or graphic design! Get involved on the [design team page][6]!
+
+## You are a Content Strategist ?
+
+We love Content, so we try to solve today's problem in producing, curating and managing content. Input from you as a Content Strategist help us to reach our goal. Contact us hello@neos.io to see how we can collaborate.
+
+## You are a coder, backend, frontend or both ?
+
+### Our stack
+
+We spend our time on the following tools, you should have a good understanding of this tooling.
+
+- PHP 7.0
+- MySQL / PostgreSQL
+- React / EmberJS 
+- GIT
+
+You can find us on [Github](https://github.com/neos) and we use it as our main issue tracker as well.
+
+### Related resources
+
+- [Development setup](https://discuss.neos.io/t/development-setup/504)
+
+## You have skills in marketing, branding or product strategy ?
+
+The Neos project is responsible for the Neos brand and we manage two product Neos CMS, our content application platform, and Flow Framework, a modern PHP based framework designed to support Neos or any complex PHP application.
+
+We are currently in the process to create the identity for the Neos brand. If you have skills in this area your are highly welcome, just ping us to have more information on the current status of the branding project.
+
+## You have other skills that you think can be useful for the community ?
+
+Who knows, some of your skills are not in this list, but can be really useful for the community. You are a project manager, an event organizer, a brewer, a team builder and joga trainer and you want to support an open source community, connect with us: **hello@neos.io** 
+
+We certainly have a something that matches your dream job and where you can help the project.
+
+## Test
+
+Your are an end user, enthusiast developer and you have some hours to help the community, let's see how you can setup Neos, test the product and report issue to the community.
+
+### Our stack
+
+We spend our time on the following tools, you should have a good understanding of this tooling.
+
+- PHPunit (unit and functional testing)
+- Behat (functional testing)
+- GIT 
+
+You can find us on [Github](https://github.com/neos) and we also use it as our main issue tracker.
+
+### Related resources
+
+- *TODO* How to setup your testing environment ?
+
+# Translate and document
+
+We use XLIFF to store the translated string for all our products. You can contribute to the translation process, by creating an account at [crowdin](https://crowdin.com/project/neos).
```

Comment:
I'd add a note that you don't need technical skills like understanding the XLIFF format (which is just an implementation detail) to contribute in this area.

"crowdin" should be uppercased